io/net/htx_nic_devices.py

```diff
@@ -148,7 +146,8 @@ def build_htx(self):
                 self.run_command("unzip master.zip")
                 self.run_command("cd HTX-master")
                 for exerciser in exercisers:
-                    self.run_command("sed -i 's/%s//g' bin/Makefile" % exerciser)
+                    self.run_command(
```

Comment:
Any reason why this script alone has long line fixes?

Reply:
ah may be i run autope8 so that did that , i think should ok

Reply:
considering we have so many files changed in this commit, better to remove these changes from this commit

Reply:
sure seems me good idea